src/README.md

```diff
@@ -0,0 +1,76 @@
+# Orchard Core Commerce
```

Comment:
Did you copy the _README.md_ from the repository root? Why? This will be a maintenance problem.

Reply:
Yes, I copied it as in OC because it becomes the root of the documentation.

Next goals will be to:
- Split the infos in multiple pages.
- Make the root Readme lighter. Ex: Just description, badges and link to doc site.

Reply:
We need to keep docs/README.md, it will be the first page of the docs site.

Reply:
Please don't resolve the conversations yourself, it makes reviewing harder as I have to look through the closed convos.

Check out [this issue](https://github.com/mkdocs/mkdocs/issues/2831#issuecomment-1204114853), looks like you can include files outside of the docs directory using the [Snippets extension](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#snippets-notation). Please try that for _docs/README.md_. I also suggest renaming it to _docs/OrchardCore.Commerce.md_ so when we include readmes for the other libraries (or any future modules) it will be consistent.

Reply:
Snippet works but we can't refer to the root Readme.

Reply:
Have you read the specific comment that I've linked in the issue? It explains where you have to put the redirect: inside the md file, not in the mkdocs.yml. You should have a file like this: https://github.com/pawamoy/duty/blob/master/docs/index.md

Reply:
Ok, done. It seems to work:
https://orchardcorecommerce.readthedocs.io/en/ag-docs/

Reply:
`Error: MD041: first-line-heading: First line in a file should be a top-level heading. Rule information: https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md041`

Reply:
Great! I've [disabled](https://github.com/DavidAnson/markdownlint#configuration) markdownlint for the file.

Reply:
F... da rules.